pyuvdata/uvdata/uvfits.py

```diff
@@ -1131,7 +1245,8 @@ def write_uvfits(
         ant_hdu.header["ARRNAM"] = self.telescope_name
         ant_hdu.header["NO_IF"] = self.Nspws
         ant_hdu.header["DEGPDY"] = self.earth_omega
-        # ant_hdu.header['IATUTC'] = 35.
+        # This is just a statically defined value
+        ant_hdu.header["IATUTC"] = 37.0
```

Comment:
I don't know why it was 35 before -- that seems to have been an error, since the difference between IAT and UTC is statically set to 37 seconds, see https://en.wikipedia.org/wiki/International_Atomic_Time

Reply:
`IATUTC` changes every leap second.  Perhaps it was 35 when the code was first written (i.e. before the two most recent leap seconds)?  The three most recent leap seconds were at the end of  2016-12-31, 2015-06-30, and 2012-06-30, so I'd guess the value in the code was last updated sometime between 2012-07-01 and 2015-06-30.  I think this is part of standard EOP products so it should probably come from there (even though it is updated far less often, and much more discretely, than UT1UTC).

Reply:
Ah, my mistake -- I was thinking TT-TAI (which is static, but not 37 seconds). We can get TAI-UTC from the IERS module in astropy, which is similar to whats done for some of the new coordinate calculation code. I'll think on this a bit and see if I can come up with an optimal  solution.